add version centos7 for compatible (#2407)
containers/buildah: contrib/buildahimage/README.md

exit
```

## Compatible with old versions

> For compatible reason, some linux distribution with kernel 3.10.0 and less will not work when use the stable image which based on fedora:32. So this centos7 branch comes for fix this.

Reviewer: ```suggestion
For compatibility reasons, some Linux distributions with a kernel version of 3.10.0 and less will not work when using the stable image that is based on fedora:32. This centos7 image can be used to work on those distributions.
```

Changes between it and stable:

> - change base image from fedora to centos:7.8.2003

Reviewer: if you change the image in the Dockerfile below, please also change this.

> - remove `--exclude container-selinux` when install buildah and fuse-overlayfs, for details pls check [here](https://bugzilla.redhat.com/show_bug.cgi?id=1806044)

Reviewer: ```suggestion
- remove `--exclude container-selinux` when installing buildah and fuse-overlayfs, for details pls check [here](https://bugzilla.redhat.com/show_bug.cgi?id=1806044)
```

> - update the sed logic of storage.conf(the final result is same, just change the method as the default file format is changed when fuse-overlayfs version is changed)

Reviewer: ```suggestion
- update the sed logic of storage.conf (the final result is the same, this change just changes the methodology)
```
I think that's what you're saying anyway.
